apply email template 적용

genie-youn opened the pull request:
#296 

지원 안내 메일에도 이메일 템플릿을 적용했습니다.
워딩과 디자인에 관한 논의는 추후에 더 필요할것 같습니다..

![kakaotalk_photo_2017-02-27-23-12-03](https://cloud.githubusercontent.com/assets/16642635/23364542/d4f9e122-fd42-11e6-9eef-6d8555a4798f.jpeg)
![kakaotalk_photo_2017-02-27-23-12-07](https://cloud.githubusercontent.com/assets/16642635/23364553/dca0d6c4-fd42-11e6-8f1f-7ad14f3abd24.jpeg)

kimtree committed:
Add a modified time param from original file

kimtree committed:
Remove deprecated twig 1.31 files

kimtree committed:
Update composer information

kimtree committed:
Change codeigniter-ss-twig library to use twig installed from composer

kimtree committed:
Optimize query to get essential columns only

genie-youn committed:
apply email template 적용

genie-youn committed:
apply email template 적용

junho85 reviewed (approved):
👍

kimtree committed:
Merge pull request #323 from yamujine/feature/#320_add_modified_time_param

Add a modified time param from original file

kimtree committed:
Merge pull request #327 from yamujine/feature/#326_mypage_apply_query_optimize

Optimize query to get essential columns only

kimtree committed:
Merge pull request #325 from yamujine/feature/#324_upgrade_twig

Feature/#324 upgrade twig

kimtree committed:
Update README.md

genie-youn committed:
Fix bug

kimtree committed:
Merge pull request #332 from yamujine/bug/#331_fix_signup_condition

Fix bug

kimtree committed:
Change PK column in ci_sessions

kimtree commented on application/libraries/Applylib.php:
```diff
@@ -17,8 +17,9 @@ public function __construct() {
+        $user_name = $this->CI->accountlib->get_user_name();
 
-        $email_html = $this->CI->twig->render('email/apply', ['artworks' => $artworks]);
+        $email_html = $this->CI->twig->render('email/apply', ['artworks' => $artworks, 'user_name' => $user_name]);
```
지난번에도 말씀드렸지만, accountlib도 다른 라이브러리 처럼 static한 function 처럼 구현이 되어야 합니다. $user_name을 파라미터로 받거나 $user_id를 넘기면 그 회원 정보를 조회해서 가져오는 식이 되는게 맞습니다.

genie-youn replied:
수정하도록 하겠습니다~

kimtree committed:
Merge pull request #334 from yamujine/feature/#333_ci_session_dup_entry

Change PK column in ci_sessions

kimtree committed:
Change Twig library to autoescape based on ‘html’ (boolean option was deprecated in 1.x twig)

kimtree committed:
Allow not_authenticate to check forced updated user session for validation is_auth

kimtree committed:
Add a hook for login check

kimtree committed:
Add custom libraries to use autocomplete

kimtree committed:
Add an image min library and optimize images

kimtree committed:
Add a phpdoc

kimtree committed:
Merge pull request #337 from yamujine/feature/#328_login_check_hook

Add a hook for login check

kimtree committed:
Remove ‘twig’ library from controllers (because it’s already defined in autoload.php)

kimtree committed:
Fix indents and wrong phpdocs

kimtree committed:
Remove landing relative features

kimtree committed:
Add a phpdoc and remove deprecated functions

mintplo committed:
Merge pull request #339 from yamujine/feature/#338_image_optimizer

Add an image min library and optimize images

kimtree committed:
Minify css

kimtree committed:
Make a verify route can be access from the person who is not authenticated

junho85 committed:
장소 - 내 작품 전시지원하기 - Filter "thumb_url" is already registered.

kimtree committed:
Add an API to CRUD exhibitions

kimtree committed:
Merge pull request #348 from yamujine/#342_add_exhibition

Add an API to CRUD exhibitions

kimtree committed:
Merge pull request #347 from yamujine/bug/#345_apply

장소 - 내 작품 전시지원하기 - Filter "thumb_url" is already registered.

kimtree committed:
Make a page for managing exhibitions

kimtree committed:
Add a monkey patching code for updating exhibition by place id

junho85 committed:
Merge pull request #349 from yamujine/feature/#342_add_exhibition_page

Make a page for managing exhibitions

junho85 committed:
전시목록 1단계
https://github.com/yamujine/artmatch/issues/350#issuecomment-302940195

junho85 committed:
전시기간이랑 작품수 사이의 바
지원하기 버튼 > 이미지
https://github.com/yamujine/artmatch/issues/350#issuecomment-303118829

kimtree committed:
Add a title column to exhibition

kimtree committed:
Modify an API and place detail function to insert and update exhibition title

kimtree committed:
Add a column to write a reason to apply

nablyy committed:
Add a view when there is no exhibition

kimtree committed:
Make a urlgenerator creates links protocol insensitive

nablyy committed:
Fix a bug

nablyy committed:
jquery3.1.1

nablyy committed:
bower

junho85 committed:
브랜치를 안따서 브랜치 땀. 지금 전시중이에요! 작업중.

junho85 committed:
지금전시중이에요 마크업

nablyy committed:
npm

nablyy committed:
return bower

kimtree committed:
Changes file permissions

kimtree committed:
Optimize png images

kimtree committed:
Apply css cleaner(minifier) for all css in assets folder

kimtree committed:
Lock npm packages (npm shrinkwrap)

nablyy committed:
setting

nablyy committed:
Merge branch 'develop' into feature/#350_exhibition_list_of_place_detail

nablyy committed:
setting

nablyy committed:
Merge branch 'feature/#350_exhibition_list_of_place_detail' of github.com:yamujine/artmatch into feature/#350_exhibition_list_of_place_detail

nablyy committed:
Merge pull request #353 from yamujine/feature/#350_exhibition_list_of_place_detail

Add a view when there is no exhibition

junho85 committed:
Merge remote-tracking branch 'origin/develop' into feature/#350_place_exhibition

# Conflicts:
#	static/dist/css/places/detail.min.css

junho85 committed:
Merge pull request #355 from yamujine/feature/#350_place_exhibition

Feature/#350 place exhibition

kimtree committed:
Merge pull request #352 from yamujine/feature/apply_reason

Add a column to write a reason to apply

kimtree committed:
Merge pull request #351 from yamujine/feature/add_exhibition_title

Feature/add exhibition title

nablyy committed:
Add script on addButton

nablyy committed:
bower

nablyy committed:
Add a button name

nablyy committed:
Change a button tag into an a tag

kimtree committed:
Update minified css and optimized css

nablyy committed:
bower

nablyy committed:
Edit commit files

nablyy committed:
Merge pull request #360 from yamujine/feature/#356_exhibition_list_in_place_detail

Feature/#356 exhibition list in place detail

kimtree committed:
Fix a bug where query can't show multiple exhibitions per place

kimtree committed:
Fix a bug that form_validation library not loaded

kimtree committed:
Add an API for showing current ongoing exhibitions per place

kimtree committed:
Remap routes

kimtree committed:
Merge pull request #361 from yamujine/feature/#344_now_exhibiting_api

Add an API for showing current ongoing exhibitions per place

kimtree committed:
Merge pull request #362 from yamujine/feature/remap_routes

Remap routes

kimtree committed:
Add autocomplete code for Twig

genie-youn committed:
Merge branch 'enhancement/#296_add_apply_notify_email_template' of https://github.com/yamujine/artmatch into enhancement/#296_add_apply_notify_email_template

genie-youn committed:
Remove dependency account library in apply library

kimtree commented:
이것도 rebase또는 머지 제대로 수정 좀 해주세요~ diff 보기가 좀 어려워서요~
